scripts/libff.sh

```diff
@@ -21,15 +23,9 @@ then
     exit 0
 fi
 
-mkdir -p $libffdir
 cd $libffdir
 mkdir -p build install
-echo "Cloning libff into ${libffdir}/src"
-git clone $libffurl src
 cd src
-echo "Fetching submodules for libff"
-git submodule init && git submodule update
-git checkout f2067162520f91438b44e71a2cab2362f1c3cab4
```

Comment:
It's still safer to have this checkout than use the latest version on the remote repo right? We don't want build fails because something changed on that repo.

Reply:
Git modules are pinned to specific commits. It's just GitHub's interface does not show it in the diff, see e.g. https://github.com/Zilliqa/scilla/tree/fix-scilla-opam-installation/deps/libff